src/distraction/DroppingBox.java

```diff
@@ -0,0 +1,104 @@
+package distraction;
+
+import com.jme3.asset.AssetManager;
+import com.jme3.bullet.BulletAppState;
+import com.jme3.bullet.control.RigidBodyControl;
+import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Spatial;
+import com.jme3.scene.shape.Box;
+import eu.opends.car.SteeringCar;
+import eu.opends.main.Simulator;
+/**
+ *
+ * @author Johnny Marek
+ */
+public class DroppingBox extends DistractionClass{
```

Comment:
let's keep some naming convention, so all distraction subclasses will be named `*Distraction`